src/main/webapp/resources/js/pages/launch/files/SampleFilesListItem.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Button, List, Radio } from "antd";
+import { grey1, grey3, grey4 } from "../../../styles/colors";
+import { SampleDetailViewer } from "../../../components/samples/SampleDetailViewer";
+import { BlockRadioInput } from "../../../components/ant.design/forms/BlockRadioInput";
+import { SPACE_XS } from "../../../styles/spacing";
+
+/**
+ * React component to display the list of files that can be run on the current
+ * pipeline for a given sample.
+ *
+ * @param {object} sample - the sample to display files for.
+ * @param {function} removeSample - function to remove the sample from the cart
+ * @param {function} updateSelectedFiles - function to call if user want to run a different set of files
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function SampleFilesListItem({
+  sample,
+  style,
+  removeSample,
+  updateSelectedFiles,
+}) {
+  const updateSelected = (e, id) => {
+    e.preventDefault();
+    updateSelectedFiles(sample, id);
+  };
+
+  return (
+    <List.Item
+      actions={[
+        <Button type="link" key={`remove`} onClick={() => removeSample(sample)}>
+          {i18n("SampleFilesListItem.remove")}
+        </Button>,
+      ]}
+      style={{
+        backgroundColor: sample.selected ? grey1 : grey3,
+        boxSizing: `border-box`,
+        borderBottom: `1px solid ${grey4}`,
```

Comment:
Use `BORDERED_LIGHT` from the `borders.js file`

Reply:
Updated in 9d2b4ca5b38bfb43c08395dffc94d037687904af